RecoBTag/Combined/python/hiRun2LegacyCSVv2Computer_cfi.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+import RecoBTag.SecondaryVertex.candidateCombinedSecondaryVertexV2Computer_cfi as sl_cfg 
+from RecoBTag.Combined.hiRun2LegacyCSVv2_trainingSettings import hiRun2LegacyCSVv2_vpset
+from RecoBTag.Combined.hiRun2LegacyCSVv2_helpers import get_vars
```

Comment:
Why do you need this get_vars function? It looks like it is not used for anything, so please remove the import. And as far as i can tell you copied two scripts from the c-tagger, i.e. the hiRun2LegacyCSVv2_trainingvars and hiRun2LegacyCSVv2_helpers files just to get this function. Can we not remove those scripts? If you need them for tests you can just use the ones in RecoBTag.CTagging